forms/FilterSelect/__tests__/FilterSelectInput-test.js

```diff
@@ -183,5 +185,33 @@ describe('FilterSelect', () => {
         expect(subject).to.eql(options)
       })
     })
+
+    context('alphabetical', () => {
+      it('lists options alphabetically asc', () => {
+        let options = [
+          { value: '1', label: 'Zim Rogers' },
+          { value: '2', label: 'Kex Perkins' },
+          { value: '3', label: 'Ben Ely' }
+        ]
+        let subject = renderIntoDocument(
+          <FilterSelect options={ options } sort={ 'asc' } />
+        ).state.filteredOptions
+
+        expect(subject).to.eql(_.sortBy(options, (option) => { return option.label }))
```

Comment:
(Subjective and not going to stop me merging but) Although it's more verbose _to write_ I feel like it would be better to assert with static values rather than using `lodash` here (and the test below).

e.g.

```js
expect(subject).to.eql([
  { value: '3', label: 'Ben Ely' },
  { value: '2', label: 'Kex Perkins' },
  { value: '1', label: 'Zim Rogers' }
])
```

I say this because it makes the test easier to read and also avoids having logic that appears mirrored from the implementation.